felix/daemon/daemon.go

```diff
@@ -410,6 +410,8 @@ configRetry:
 		v3Client,
 		dpDriver,
 		failureReportChan)
+	// Start communicating with the dataplane driver.
+	dpConnector.Start()
```

Comment:
Noting here (from @fasaxc)
Starting the DP Connector this early is thread-unsafe as the connector is still only half-built.
Need to mitigate any data races here